website/content/docs/configuration/client.mdx

```diff
@@ -247,15 +247,17 @@ chroot as doing so would cause infinite recursion.
   access files only within the [task working directory].
 
 - `max_stale` `(string: "87600h")` - This is the maximum interval to allow "stale"
-  data. By default, only the Consul leader will respond to queries. Requests to
-  a follower will forward to the leader. In large clusters with many requests,
-  this is not as scalable. This option allows any follower to respond to a query,
-  so long as the last-replicated data is within this bound. Higher values result
-  in less cluster load, but are more likely to have outdated data.
+  data. If `max_stale` is not set, only the Consul leader will respond to queries, and
+  requests that reach a follower will forward to the leader. In large clusters with
+  many requests, this is not as scalable. This option allows any follower to respond
+  to a query, so long as the last-replicated data is within this bound. Higher values
+  result in less cluster load, but are more likely to have outdated data. This default
+  of 10 years (`87600h`) matches the default Consul configuration.
```

Comment:
Ha I hadn't done the math before. I guess that's a valid way to implement "indefinitely" :sweat_smile: